src/components/clusters/ClusterDetailsMultiRegion/components/SecurityGroups/SecurityGroupsNoChangeAlert.tsx

```diff
@@ -4,26 +4,37 @@ import { Alert, AlertActionLink } from '@patternfly/react-core';
 
 import links from '~/common/installLinks.mjs';
 
-const SecurityGroupsNoChangeAlert = ({ isRosa }: { isRosa?: boolean }) => (
-  <Alert
-    variant="info"
-    isInline
-    title="You cannot add or edit security groups to the machine pool nodes after they are created."
-    actionLinks={
-      <>
-        <AlertActionLink
-          component="a"
-          href={isRosa ? links.ROSA_SECURITY_GROUPS : links.OSD_SECURITY_GROUPS}
-          target="_blank"
-        >
-          View more information
-        </AlertActionLink>
-        <AlertActionLink component="a" href={links.AWS_CONSOLE_SECURITY_GROUPS} target="_blank">
-          AWS security groups console
-        </AlertActionLink>
-      </>
-    }
-  />
-);
+const SecurityGroupsNoChangeAlert = ({
+  isRosa,
+  isHypershift,
+}: {
+  isRosa?: boolean;
+  isHypershift: boolean;
```

Comment:
this prop should be optional, because:
- its logic is a derivative of `isRosa`, which is optional
- it's a boolean, which in javascript means it's naturally coerced as such in conditionals (so we get the benefit of having a default value of 'false')